engine/src/main/java/org/terasology/config/Config.java

```diff
@@ -65,6 +66,7 @@
  * Terasology user config. Holds the various global configuration information that the user can modify. It can be saved
  * and loaded in a JSON format.
  */
+@API
```

Comment:
I'm not sure if it is safe giving modules access to the root `Config` object? Wouldn't that in theory allow modules to read the "security" block which includes some certificate stuff? Something deeper like only the metrics subset seems fine, but I dunno if that's problematic